src/client/mds_client_base.h

```diff
@@ -107,7 +108,8 @@ class MDSClientBase {
                   const UserInfo_t& userinfo,
                   OpenFileResponse* response,
                   brpc::Controller* cntl,
-                  brpc::Channel* channel);
+                  brpc::Channel* channel,
+                  const OpenFileContext_t& cont = {});
```

Comment:
`OpenFileContext` should not be a default parameter.

Another suggestion is we can encapsulate fileinfo/userinfo into `OpenFileContext`